<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWidget</class>
 <widget class="QWidget" name="MainWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>582</width>
    <height>160</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Desktop Entry</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="0">
    <widget class="QLabel" name="iconInfo">
     <property name="text">
      <string>Icon</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QToolButton" name="iconSelectButton">
     <property name="text">
      <string>...</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1" colspan="2">
    <widget class="QCommandLinkButton" name="createButton">
     <property name="text">
      <string>Create Desktop Entry</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QToolButton" name="exeSelectButton">
     <property name="text">
      <string>...</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="exeInfo">
     <property name="text">
      <string>Executable</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLineEdit" name="iconEdit"/>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="iconPreview">
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="exeEdit"/>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="nameInfo">
     <property name="text">
      <string>Name</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="nameEdit"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
